.gitignore

```diff
@@ -1,2 +1,3 @@
 /target
+Cargo.lock
```

Comment:
Committing this was actually intentional to make the on-target tests reproducible, though I now realize that it isn't effective since CI isn't adding `--locked` to the compiler arguments.

Reply:
I'll revert it!